index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
         </li>
       </ul>
     </section>
+    <!--part II-->
+    <section class="workSection">
+      <h1 class="ptitle t2">My Recent Works</h1>
```

Comment:
- [ ] Good job so far 👍 For best semantic practices your site should have only one main `h1` heading tag. Kindly change this into an `h2` tag.

Reply:
Im working on it right away! 
just finished this part